example.env

```diff
@@ -1,8 +1,11 @@
 # copy and rename this file to '.env' to take effect
 BACKEND_PORT=3000
+ADMINER_PORT=1337
```

Comment:
NIT: If we're going to order by block, can we break them up in to labelled sections and alpha-order inside the blocks?